src/main/java/sysc4806/project24/mini_survey_monkey/Constant.java

```diff
@@ -2,6 +2,8 @@
 
 public final class Constant {
 
+    public static final String GUEST_USERNAME = "guest";
+
     public final class CookieKey {
```

Comment:
Do you think it’s a good idea to make CookieKey a static nested class to avoid unnecessary references to the outer Constant class?